src/reducers/reducers-cities.js

```diff
@@ -0,0 +1,23 @@
+import { FETCH_CITY } from "../actions";
+
+const cityReducer = function (state = [], action) {
```

Comment:
This replaces the current city, but we want to be able to add several cities to the list to compare